itchef/cookbooks/cpe_vfuse/resources/darwin.rb

```diff
@@ -74,4 +79,19 @@ def templates
       end
     end
   end
+
+  def configure
+    template_dir = node['cpe_vfuse']['template_dir']
+    return if template_dir.nil?
+    config = {
+      'template_dir' => template_dir,
+    }
+    file '/usr/local/vfuse/bin/config.json' do
```

Comment:
Can you add an only_if Dir.exist?('/usr/local/vfuse/bin') here? Not everyone is going to be installing vfuse into /usr/local/vfuse/bin.